store/siren-action-behavior.js

```diff
@@ -184,7 +189,54 @@ D2L.PolymerBehaviors.Siren.SirenActionBehaviorImpl = {
 					return window.D2L.Siren.EntityStore.update(url.href, token, entity);
 				});
 			});
-	}
+	},
+
+	_getToken: function(token) {
```

Comment:
We already import the `entity-store` into this file, I think we should just use the function from there rather than duplicate it (unless you had to change something?)

Reply:
You're totally right! Good call